### PR TITLE
Added a couple of articles/prepositions that were missing

### PR DESCRIPTION
**Pull request by olgabanis**
Simple documentation errata.

### CODE REVIEW

**Comment by facebook-github-bot-4** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: docs/Images.md [hunk 1]
```diff
@@ -34,15 +34,15 @@ And `button.js` code contains
 <Image source={require('./img/check.png')} />
 ```
 
-Packager will bundle and serve the image corresponding to device's screen density, e.g. on iPhone 5s `check@2x.png` will be used, on Nexus 5 – `check@3x.png`. If there is no image matching the screen density, the closest best option will be selected.
+Packager will bundle and serve the image corresponding to the device's screen density, e.g. on iPhone 5s `check@2x.png` will be used, on Nexus 5 – `check@3x.png`. If there is no image matching the screen density, the closest best option will be selected.
```
Comment (davidaurelio) [inline docs/Images.md]: I’m not a native speaker, but wouldn’t “to the screen density of the device” be even better.

File: docs/Images.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ iOS saves multiple sizes for the same image in your Camera Roll, it is very impo
 
 *In the browser* if you don't give a size to an image, the browser is going to render a 0x0 element, download the image, and then render the image based with the correct size. The big issue with this behavior is that your UI is going to jump all around as images load, this makes for a very bad user experience.
 
-*In React Native* this behavior is intentionally not implemented. It is more work for the developer to know the dimensions (or aspect ratio) of the remote image in advance, but we believe that it leads to a better user experience. Static images loaded from the app bundle via the `require('./my-icon.png')` syntax *can be automatically sized* because their dimensions are available immediately at the time of mounting.
+*In React Native* this behavior is intentionally not implemented. It is more work for the developer to know the dimensions (or aspect ratio) of the remote image in advance, but we believe that it leads to a better user experience. Static images loaded from the app and bundled via the `require('./my-icon.png')` syntax *can be automatically sized* because their dimensions are available immediately at the time of mounting.
```
Comment (davidaurelio) [inline docs/Images.md]: This one was actually legit.

**Comment by davidaurelio** [comment]
Thank you for taking the time to make our documentation better. One of your changes was actually correct. Could you revert that one, please?

File: docs/Images.md (hunk 1 above)
Comment (olgabanis) [inline docs/Images.md]: Hey David, it can go both ways. I didn't want to get into changing the wording and grammar.

**Comment by olgabanis** [comment]
Hey David, thanks for letting me know that one of my changes for actually correct. The other one was correct, too. Take care

**Comment by davidaurelio** [comment]
@olgabanis oh wow, I’m sorry. That was really misleading. I wanted to say that one of your edits was actually unnecessary – the one that I was commenting on. I just messed up the comment, sorry about that.

File: docs/Images.md (hunk 1 above)
Comment (davidaurelio) [inline docs/Images.md]: Feel free to go all the way, or keep it like this – your call :-)

**Comment by JoelMarcey** [comment]
cc @lacker @hramos

**Comment by lacker** [comment]
Hey @olgabanis thanks for sending these fixes! I think I get what David was attempting to say here - the "app bundle" is what that sentence is referring to, using bundle as a noun. So I just tweaked the PR to have the other three changes. Thanks again - I'll ship this now

**Comment by lacker** [comment]
@facebook-github-bot shipit

**Comment by olgabanis** [comment]
Thanks @lacker and sorry for letting this drag on, I thought it was resolved. Cheers

**Pull request closed without merging**